graylog2-web-interface/src/components/dashboard/Dashboard.jsx

```diff
@@ -68,16 +68,14 @@ const Dashboard = React.createClass({
     return (
       <li className="stream">
         <h2>
-          <LinkContainer to={Routes.dashboard_show(this.props.dashboard.id)}>
-            <a><span ref="dashboardTitle">{this.props.dashboard.title}</span></a>
-          </LinkContainer>
+          <Link to={Routes.dashboard_show(this.props.dashboard.id)}>{this.props.dashboard.title}</Link>
```

Comment:
`Link` is not imported in the file. Since 2.4 uses a different react-router version than 3.0, I would keep the old `LinkContainer` component and play it safe.

Reply:
Good point! 👍